lib/rubocop/cop/rails/active_record_aliases.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks that ActiveRecord aliases are not used.
+      #
+      # @example
+      #   #bad
+      #   Book.update_attributes!(author: 'Alice')
+      #
+      #   #good
+      #   Book.update!(author: 'Alice')
+      class ActiveRecordAliases < Cop
+        MSG = 'Use `%<prefer>s` instead of `%<current>s`.'.freeze
+
+        ALIASES = {
+          update_attributes: :update,
+          update_attributes!: :update!
+        }.freeze
+
+        def on_send(node)
+          ALIASES.each do |bad, good|
+            next unless node.method?(bad)
+
+            add_offense(node,
+                        message: format(MSG, prefer: good, current: bad),
+                        location: :selector,
+                        severity: :warning)
+            break
```

Comment:
This `break` did not seem to be necessary.

Reply:
It will prevent checking the remaining aliases if an offense has already been found for the current node.

Reply:
I see. However, it seems that it will not change as behaviour whether or not `break` is present when inspecting the following code.

```ruby
# frozen_string_literal: true

model.update_attributes(foo: bar)
model.update_attributes!(foo: bar)
```

Reply:
The behavior is the same, but it reduces unnecessary checking.

When `node.method?(bad)` above is true, then we know that the node matches the current method name in the `ALIASES` loop. It is impossible for more than one name to match, so there is no need to continue the loop to check the other names in `ALIASES`.

Reply:
I get it, I understand! Thank you for your explanation 🙏